queries.sql

```diff
@@ -14,4 +14,33 @@ SELECT name FROM animals WHERE neutered IS TRUE;
 
 SELECT name FROM animals WHERE name NOT IN ('Gabumon');
 
-SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
```

Comment:
- [ ] You should verify that change was undone after rollback as requested in the project requirements

>Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and **verify that species columns went back to the state before tranasction**.